src/Symfony/Component/String/Tests/UuidTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\String\Uuid;
+
+class UuidTest extends TestCase
+{
+    private const AN_UUID_V1 = 'd9e7a184-5d5b-11ea-a62a-3499710062d0';
+    private const AN_UUID_V4 = 'd6b3345b-2905-4048-a83c-b5988e765d98';
+
+    protected function setUp(): void
+    {
+        if (!\function_exists('uuid_is_valid')) {
+            $this->markTestSkipped('The uuid extension is required');
```

Comment:
`@requires extension uuid` on the class instead

Reply:
But it could be provided by the polyfill, that's why I did that

Reply:
IIRC, there is a `@requires function` in PHPUnit too